spec/requests/shopping_lists_spec.rb

```diff
@@ -772,8 +772,17 @@
 
       context 'when the shopping list exists' do
         let!(:shopping_list) { create(:shopping_list, game:) }
+        let!(:shopping_list_id) { shopping_list.id }
 
         context "when this is the game's last regular shopping list" do
+          let!(:aggregate_list_id) { game.aggregate_shopping_list.id }
```

Comment:
There would've been a few ways to skin this cat, including `shopping_list.aggregate_list_id`. I didn't see too many advantages of one approach over another.